hera_cal/tests/test_tempcal.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 the HERA Project
+# Licensed under the MIT License
+
+from __future__ import print_function, division, absolute_import
+
+import pytest
+import numpy as np
+import sys
+import os
+import glob
+import six
+import copy
+from pyuvdata import UVData
+
+from .. import utils, io, apply_cal
+from ..tempcal import TempCal
+from ..data import DATA_PATH
+
+
+def test_tempcal():
+    # load data
+    dfiles = sorted(glob.glob(os.path.join(DATA_PATH, "zen.2458043.*.xx.HH.XRAA.uvh5")))
+    T = TempCal(dfiles)
+    autos = [bl for bl in T.bls if bl[0] == bl[1]]
```

Comment:
This will also include cross-polarizations, which I don't think you want. Dunno if that file has any, but best to be explicit.